### PR TITLE
3.4.10. Adpated to pvr addon api 5.2.1 (3.4.9 was just to ake it compile again).

### PR DESCRIPTION
**Pull request by ksooo**
@Jalle19 we have to cleanup to actually benefit from the API change.

### CODE REVIEW

File: src/tvheadend/Subscription.cpp [hunk 1]
```diff
@@ -173,12 +173,12 @@ void Subscription::SendUnsubscribe(void)
...
 }
 
-bool Subscription::SendSeek(int time)
+bool Subscription::SendSeek(double time)
 {
   /* Build message */
   htsmsg_t *m = htsmsg_create_map();
   htsmsg_add_u32(m, "subscriptionId", GetId());
-  htsmsg_add_s64(m, "time",           static_cast<int64_t>(time * 1000LL));
+  htsmsg_add_s64(m, "time",           time * 1000LL);
```
Comment (Jalle19) [inline src/tvheadend/Subscription.cpp]: Why no cast anymore? `htsmsg_add_s64` still expects an `int64_t`, not a double, right?

Comment (ksooo) [inline src/tvheadend/Subscription.cpp]: Good point. Will revert this line.

**Comment by ksooo** [comment]
@Jalle19 addressed your review comment. Good to go now?

**Comment by Jalle19** [comment]
+1

**Pull request merged**